e2e/testdata/fn-render/validator-mutates-resources/Kptfile

```diff
@@ -4,6 +4,6 @@ metadata:
   name: app
 pipeline:
   validators:
-    - image: gcr.io/kpt-fn/set-labels:v0.1.4 # mutator function is being used as validator
+    - image: gcr.io/kpt-fn/set-labels:v0.1.4
```

Comment:
render formats the `Kptfile` that doesn't preserve comments causing this test to fail, so removed the comment from `Kptfile`.

Reply:
Is this a known bug? Is it being tracked?

Reply:
There are a few bugs related to `kptfile` formatting etc, but it's a good idea to track it explicitly. File https://github.com/GoogleContainerTools/kpt/issues/2301